specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -401,7 +448,7 @@
           "LiveEvents"
         ],
         "summary": "Reset Live Event",
-        "description": "Resets an existing Live Event.",
+        "description": "Resets an existing live event.",
```

Comment:
Lets expand on this one to include what happens during a reset operation, and in what situations one should call Reset on a live event.

Reply:
Resets an existing live event. All live outputs for the live event are deleted and the live event is stopped and started again. All assets used by the live outputs and streaming locators created on these assets are unaffected.